### ISSUE TITLE
' is being replaced with ? 

### ISSUE THREAD

**Issue by Nix-Starlight** [CoagulateSL/GPHUD#131]
When using the following code 

String response=gsGetChoice(CALLER,"Edit this!",["ok"]);

If replacing with a phrase in Edit This! with 'these' around the word, it becomes ?

String response=gsGetChoice(CALLER,"I will use 'this' as an example",["ok"]);

I will use ?this? as an example

**Comment by dazedwolf** [comment]
You're probably experiencing the same issue I had. If you're typing your script in an external platform you might be writing ‘ instead of '. 

The language currently doesn't support the use of the ‘ character (left single quotation mark) and you'd have to use the apostrophe character ( ' ).

Using something like notepad++ or visual studio automatically replaces the left single quotation mark with the apostrophe, so it shouldn't happen.

**Comment by iprice** [comment]
Yeah, don't use Microsoft Office or any other office suite for writing scripts ; these applications tend to replace single quotes, double quotes and even hyphens with "smart" versions of the same.  Which basically wont work in many places.

That said this probably should work and I've long had limitations hanging around in the character set.  Not that I really want character names to become something most people can only copy paste, so the limitation has its uses.

But one of these years I should probably do this properly.  Maybe.

**Comment by Nix-Starlight** [comment]
> Yeah, don't use Microsoft Office or any other office suite for writing scripts ; these applications tend to replace single quotes, double quotes and even hyphens with "smart" versions of the same. Which basically wont work in many places.
> 
> That said this probably should work and I've long had limitations hanging around in the character set. Not that I really want character names to become something most people can only copy paste, so the limitation has its uses.
> 
> But one of these years I should probably do this properly. Maybe.

I wasn't using externals. I don't tend to in general and don't have Word or it's like on my laptop at all. I don't remember what I using this for since it was awhile ago.
